src/applications/vaos/services/mocks/wellHive/patientBasicReferralDetails.json

```diff
@@ -0,0 +1,19 @@
+{
+  "data": {
+    "referral": {
```

Comment:
we can add details as part of the referrals array and add multiple referrals assuming api returns multiple referrals
```suggestion
    "referrals": [
      {
      "id": "mock123",
      "typeOfCare": "Primary Care",
      "referralNumber": 123456,
      "startDate": "2021-01-01",
      "expirationDate": "2021-02-01",
      "referringFacility": "White River VA Clinic, VT",
      "phoneNumber": "555-555-5555",
      "detailsShared": {
        "preferredProvider": "Dr. Smith",
        "preferredLocation": "123 Main St, Springfield, IL 62704",
        "preferredFacility": "Springfield VA Clinic",
        "bookNotes": "Patient has a history of back pain"
      }
    },
    {
      "id": "mock456",
      "typeOfCare": "Audiology",
      "referralNumber": 67890,
      "startDate": "2024-11-01",
      "expirationDate": "2024-12-01",
      "referringFacility": "White River VA Clinic, VT",
      "phoneNumber": "234-567-8900",
      "detailsShared": {
        "preferredProvider": "Dr. Smith",
        "preferredLocation": "123 Main St, Springfield, IL 62704",
        "preferredFacility": "Springfield VA Clinic",
        "bookNotes": "Patient has a history of back pain"
      }
    }
  ]
```